packages/wallets/src/features/cashier/modules/DepositCrypto/components/DepositCryptoCurrencyDetails/__tests__/DepositCryptoCurrencyDetails.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useActiveWalletAccount } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import DepositCryptoCurrencyDetails from '../DepositCryptoCurrencyDetails';
+
+jest.mock('@deriv/api');
+
+describe('DepositCryptoCurrencyDetails', () => {
+    const mockData = {
+        currency_config: {
+            display_code: 'BTC',
+            name: 'Bitcoin',
+        },
+    };
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render with correct currency details', () => {
+        (useActiveWalletAccount as jest.Mock).mockReturnValue({ data: mockData });
+
+        render(<DepositCryptoCurrencyDetails />);
+
+        expect(screen.getByText(`Send only Bitcoin (BTC) to this address`)).toBeInTheDocument();
+    });
+
+    it('should render with default text when data is unavailable', () => {
+        (useActiveWalletAccount as jest.Mock).mockReturnValue({});
+
+        render(<DepositCryptoCurrencyDetails />);
+
+        expect(screen.getByText('Send only () to this address')).toBeInTheDocument();
```

Comment:
actually - I do not belive we should test that something misbehaves. 
What I mean is that: 'Send only () to this address' is not correct text to begin with, so testing that this text is dispalyed, is like documenting that system should behave incorrectly;

Reply:
in ideal scenario - this test should be written during the component building time, so then that scenario would be uncovered during component build phase and should be fixed or explicitly handled

Reply:
Updated to a more logical approach